frontend/src/v5/ui/routes/viewer/tickets/ticketsForm/commentsPanel/comment/commentNonMessageContent/commentNonMessageContent.component.tsx

```diff
@@ -49,9 +53,17 @@ export const CommentNonMessageContent = ({
 				/>
 			)}
 			{images.length > 1 && (
```

Comment:
This is now the same as the condition for when `images.length === 1`, so you can combine it into a `images.length >= 1` condition

Reply:
If I get what you mean (i.e. both when I have `images.length === 1` and `images.length > 1` I render `CommentImages` component), I cannot because, as for the old design, when you have just one image, that should be displayed before the author and the reply, whereas multiple images should be displayed after the latter